### PR TITLE
add Acorn app =D

### PR DESCRIPTION
**Pull request by Connoropolous**
ready for merge!

Tested it with the manual file based install
Screen Shot 2020-02-25 at 4 26 42 PM

### CODE REVIEW

File: index.json [hunk 1]
```diff
@@ -6,5 +6,13 @@
     "icon": null,
     "screenshot": "https://raw.githubusercontent.com/holochain/peer-chat/develop/doc/screen.png",
     "description": "A simple chat app designed to get new users up, running and developing on Holochain"
+  },
+  {
+    "name": "Acorn Project Management",
```
Comment (samcooley) [inline index.json]: Acorn

File: index.json [hunk 2]
```diff
@@ -6,5 +6,13 @@
...
+    "directory": "acorn",
+    "github": "https://github.com/h-be/acorn-hc",
+    "icon": null,
```
Comment (samcooley) [inline index.json]: https://github.com/h-be/acorn-ui/blob/master/dist/logo/acorn-logo-desktop-512px%402x.png (or a smaller icon image)

File: index.json [hunk 3]
```diff
@@ -6,5 +6,13 @@
...
+    "screenshot": "...",
```
Comment (samcooley) [inline index.json]: https://i.imgur.com/vyvdfyn.png
or 
https://i.imgur.com/2LreFT4.png

File: index.json [hunk 4]
```diff
@@ -6,5 +6,13 @@
...
+    "description": "..."
```
Comment (samcooley) [inline index.json]: Acorn is a way of organizing intended Goals and Outcomes on a tree.  You can think of it as a virtual whiteboard for collaborative planning.

**Review by samcooley — CHANGES_REQUESTED** [review]
comments suggesting changes added

**Review by zippy — APPROVED** [review]

**Review by lucksus — APPROVED** [review]
:tada: :tada: :tada:

**Pull request merged**